### ISSUE TITLE
Get target temperature - continued

### ISSUE THREAD

**Issue by jnxxx** [MTrab/danfoss_ally#23]
Don't give up... 🥺

I see the integration reads and writes `temp_set` in the API.
I too can conclude it reads the Home setpoint and seem to write the Manual setpoint.

I have not seen any documentation from Danfoss on what `temp_set` is meant to represent. Should it read/write the setpoint for the current mode? In any case it seems strange it reads and writes different setpoints.

Locally I have made a change to it, so instead it uses `manual_mode_fast`. Now it works reliably, I believe, but of course always for the manual setpoint.

Question is how you intend the climate control to work?

Should it always show and set the manual setpoint, then just change `temp_set` to `manual_mode_fast` (not tested with radiator thermostats).

Should it show and set the setpoint for the current preset mode, then probably use:
-   Manual: `manual_mode_fast`
-   Home: `at_home_setting`
-   Away: `leaving_home_setting`
-   Pause: `pause_setting`
-   Holiday: `holiday_setting`

when reading and writing, depending the the current preset mode.

The app can set these setpoints independent of the preset mode.
To allow for this, maybe implement a service call (I think it is [this](https://developers.home-assistant.io/docs/device_automation_action)).
Alternatively, a full climate control for each....

**Floor temperature sensor**
I also extended my local version with a floor temperature sensor, read from `MeasuredValue` if 'floor_sensor' is true.
To also be able to read and write the min and max setpoints, it should probably have its own climate control with `TARGET_TEMPERATURE_RANGE` rather than `TARGET_TEMPERATURE`.
I tried if one climate control can handle both, but it doesn't seem to work properly.

I can share what I have, but I had to place pydanfossally locally also, so not ideal to make a pull request.

**Comment by jnxxx** [comment]
I have now forked it and placed my suggestions (so far) here: https://github.com/jnxxx/danfoss_ally

Improvements:

- Reading and writing setpoints using: `manual_mode_fast`, `at_home_setting`, `leaving_home_setting`, `pause_setting`, `holiday_setting` depending on the preset mode, rather than using `temp_set` as before.
It seems to work, so far.
 
- Holiday preset mode added.
 It is not predefined, but you are allowed to add custom.
 
- A change in hvac mode, preset mode or setpoint temperature applies faster in the UI.
Initiates a read for a single device after writing.
Maybe poll time can then be lowered a bit, 15 seconds is quite aggressive I think.
 
- Floor temperature sensor
 _Note:_ The app shows the floor temperature (when present) in the overview, and the room temperature on the details page. That is somewhat confusing, I think. Especially when it doesn't indicate it which is which.
 
- Fix for setmode issue (despite version v1.0.7 I experienced the issue, maybe something went wrong in publishing this version?)
